event/src/lib.rs

```diff
@@ -1,14 +1,25 @@
 //! # Ceramic Event
 //! Implementation of ceramic event protocol, with appropriate compatibilility with js-ceramic
 #![deny(missing_docs)]
-mod args;
-pub mod bytes;
-mod deterministic_init_event;
-mod event;
-pub mod unvalidated;
+mod bytes;
+mod signed_event;
+mod unvalidated;
+mod validated;
```

Comment:
Can we separate out the validated module changes from the builder changes into their own PR? Then we can review just the builder changes. I do not have a good grasp on how validation will work yet as there are many dynamic dependencies required for performing event validation that is not clear how we should structure that process yet. However the builder bits are valuable now both for testing and general debugging.